tasks/copy.js

```diff
@@ -0,0 +1,21 @@
+var gulp = require('gulp');
+var copy = require('gulp-contrib-copy');
+var srcDir = 'src';
```

Comment:
We should find a way to avoid this duplication. What happens if you change the `src` path? You should change all occurrences and this is not very convenient.